Fabric.Identity.API/scripts/tests/readme.md

```diff
@@ -0,0 +1,37 @@
+# Running CatalystDosIdentity Functional tests
+
+## Run Against Remote Setup
+To run against a remote identity server, you will need to point the tests to the identity url and pass in the installer secret. 
+
+Sample call:
+```
+./CatalystDosIdentity.functional.tests.ps1 -identityUrl "https://server/identity" -installerSecret "secretKey"
+```
+
+## Run Against Local Docker Setup
+## Setup Docker
+Docker needs to be up and running and set to run Linux containers.
+
+Run the start-identity.sh script to start the required docker images.
```

Comment:
do the .sh files need to be run in BASH and the .ps1 in powershell?   if so we should note it.